Source/Scene/ImplicitTileset.js

```diff
@@ -0,0 +1,249 @@
+import Check from "../Core/Check.js";
+import clone from "../Core/clone.js";
+import defined from "../Core/defined.js";
+import Resource from "../Core/Resource.js";
+import RuntimeError from "../Core/RuntimeError.js";
+import has3DTilesExtension from "./has3DTilesExtension.js";
+import ImplicitSubdivisionScheme from "./ImplicitSubdivisionScheme.js";
+
+/**
+ * An ImplicitTileset is a simple struct that stores information about the
+ * structure of a single implicit tileset. This includes template URIs for
+ * locating resources, details from the implicit root tile (bounding volume,
+ * geometricError, etc.), and details about the subtrees (e.g. subtreeLevels,
+ * subdivisionScheme).
+ *
+ * @alias ImplicitTileset
+ * @constructor
+ *
+ * @param {Resource} baseResource The base resource for the tileset
+ * @param {Object} tileJson The JSON header of the tile with the 3DTILES_implicit_tiling extension.
+ * @param {MetadataSchema} [metadataSchema] The metadata schema containing the implicit tile metadata class.
+ * @private
+ * @experimental This feature is using part of the 3D Tiles spec that is not final and is subject to change without Cesium's standard deprecation policy.
+ */
+export default function ImplicitTileset(
+  baseResource,
+  tileJson,
+  metadataSchema
+) {
+  var extension = tileJson.extensions["3DTILES_implicit_tiling"];
+  //>>includeStart('debug', pragmas.debug);
+  Check.typeOf.object(
+    'tileJson.extensions["3DTILES_implicit_tiling"]',
+    extension
+  );
+  //>>includeEnd('debug');
+
+  /**
+   * The base resource for the tileset. This is stored here as it is needed
+   * later when expanding Implicit3DTileContents so tile URLs are relative
+   * to the tileset, not the subtree file.
+   *
+   * @type {Resource}
+   * @readonly
+   * @private
```

Comment:
Looks like implicit tiling classes are missing the `@memberof`  JSDoc

Reply:
`@memberof` is only needed for getters/setters I believe